projects/layout/components/form/form.directive.ts

```diff
@@ -39,20 +45,23 @@ class TuiFormStyles {}
     standalone: true,
     selector: '[tuiForm]',
     providers: [
-        tuiButtonOptionsProvider(TuiForm),
-        tuiNotificationOptionsProvider(TuiForm),
+        projectSize(TUI_BUTTON_OPTIONS, (size) => size),
+        projectSize(TUI_NOTIFICATION_OPTIONS, (size) => size),
         projectSize(TUI_HEADER_OPTIONS, (size) => HEADER_SIZE[size]!),
```

Comment:
What if we drop type of `HEADER_SIZE` and add `as const`? We can then change type in projectSize to `TuiHandler<TuiFormOptions['size'], string>` and get rid of `!`?